src/cyclus.h

```diff
@@ -59,4 +59,9 @@
 #include "toolkit/symbolic_functions.h"
 #include "toolkit/timeseries.h"
 
+// Undefines isnan from pyne
+#ifdef isnan
```

Comment:
The right thing to do would be to make PyNE C++11 only and then remove the ifdef there.  Until that time, though...